src/java/org/apache/cassandra/config/SmallestDurationMilliseconds.java

```diff
@@ -54,4 +61,9 @@ public static SmallestDurationMilliseconds inMilliseconds(long milliseconds)
     {
         return new SmallestDurationMilliseconds(milliseconds, TimeUnit.MILLISECONDS);
     }
+
+    public static SmallestDurationMilliseconds inDoubleMilliseconds(double milliseconds)
```

Comment:
nit: The "Double" part isn't strictly necessary, but it's not harmful either ;)

Reply:
I will keep it for now